tests/tests/tier1/hirte-proxy-service/main.fmf

```diff
@@ -0,0 +1,6 @@
+summary: Tests for the proxy service feature of hirte to resolve dependencies on other nodes
+
+tier: 1
```

Comment:
Is there a reason why not to have basic proxy test in tier0? I think that in tier1 there should be only more advanced or corner case tests, but all important functionality needs to be tests within tier 0

Reply:
Not really. I didn't really think about it, to be honest. It makes sense to group it like you explained. I'll move it to tier 0.

Reply:
+1